install_config/persistent_storage/topics/glusterfs_security_selinux.adoc

```diff
@@ -1,16 +1,24 @@
 By default, SELinux does not allow writing from a pod to a remote {gluster}
-server. To enable writing to {gluster} volumes with SELinux on, run the
+server. To enable writing to {gluster} volumes with SELinux on, type the
 following on each node running GlusterFS:
 
 [source,bash]
 ----
 $ sudo setsebool -P virt_sandbox_use_fusefs on <1>
+$ sudo setsebool -P virt_use_fusefs on
 ----
-<1> The `-P` option makes the bool persistent between reboots.
+<1> The `-P` option makes the bool persistent between reboots. 
 
 [NOTE]
 ====
 The `virt_sandbox_use_fusefs` boolean is defined by the *docker-selinux*
 package. If you get an error saying it is not defined, ensure that this
 package is installed.
 ====
+
+[NOTE]
+====
+If using Atomic Host instead of RHEL, the SELinux booleans are cleared upon 
+version upgrades, forcing them to set them manually. Failing to set both 
```

Comment:
maybe s/forcing them to set them manually./When you upgrade Atomic Host, you must set these boolean values again.